morphdom not found because phoenix_html imported w/o deps path

### Environment

* Elixir version (elixir -v): 1.12.1 on OTP 24
* Phoenix version (mix deps): github master
* Phoenix LiveView version (mix deps): github master
* NodeJS version (node -v): 14.17.0
* NPM version (npm -v): 6.14.13
* Operating system: Windows 10
* Browsers you attempted to reproduce this bug on (the more the merrier): N/A
* Does the problem persist after removing "assets/node_modules" and trying again? Yes/no: Yes

### Actual behavior

I tried pulling the master branch in to test out the [webpack 5 build](https://github.com/phoenixframework/phoenix/pull/4337).  I get this error when starting the app.  I noticed in the changelog there is a bug fix with a description quite similar to what I'm experiencing but I can't see an issue or PR related.

I found [this](https://stackoverflow.com/questions/43037590/field-browser-doesnt-contain-a-valid-alias-configuration) answer on stackoverflow which says to replace `import "phoenix_html"` `import "../../deps/phoenix_html"`.  I had to make this change and `cd deps/phoenix_live_view/assets` and run `npm i` for it to work.

I feel like I've missed a step somewhere...?

```
Module not found: Error: Can't resolve 'morphdom' in 'C:\Users\simon\code\logacy\deps\phoenix_live_view\assets\js\phresolve 'morphdom' in 'C:\Users\simon\code\logacy\deps\phoenix_live_view\assets\js\phoenix_live_view'
  using description file: C:\Users\simon\code\logacy\deps\phoenix_live_view\assets\package.json (relative path: ./js/phoenix_live_view)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      C:\Users\simon\code\logacy\deps\phoenix_live_view\assets\js\phoenix_live_view\node_modules doesn't exist or is not a directory
      C:\Users\simon\code\logacy\deps\phoenix_live_view\assets\js\node_modules doesn't exist or is not a directory
      C:\Users\simon\code\logacy\deps\phoenix_live_view\assets\node_modules doesn't exist or is not a directory
      C:\Users\simon\code\logacy\deps\phoenix_live_view\node_modules doesn't exist or is not a directory
      C:\Users\simon\code\logacy\deps\node_modules doesn't exist or is not a directory
      C:\Users\simon\code\logacy\node_modules doesn't exist or is not a directory
      C:\Users\simon\code\node_modules doesn't exist or is not a directory
      C:\Users\simon\node_modules doesn't exist or is not a directory
      C:\Users\node_modules doesn't exist or is not a directory
      C:\node_modules doesn't exist or is not a directory
 @ ../deps/phoenix_live_view/assets/js/phoenix_live_view/view.js 5:0-35 316:20-28 495:24-32 521:20-28 816:8-24
 @ ../deps/phoenix_live_viiex(1)> ew/assets/js/phoenix_live_view/live_socket.js 76:0-26 357:19-23
 @ ../deps/phoenix_live_view/assets/js/phoenix_live_view/index.js 9:0-39 10:0-22
 @ ./js/app.js 28:0-47 61:21-31

webpack 5.37.1 compiled with 1 error in 8635 ms
```

### Expected behavior

app runs

--- a/package.json
+++ b/package.json
@@ -13,6 +13,9 @@
     "type": "git",
     "url": "git://github.com/phoenixframework/phoenix_live_view.git"
   },
+  "dependencies": {
+    "morphdom": "2.6.1"
+  },
   "files": [
     "README.md",
     "LICENSE.md",

